ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -35,6 +36,7 @@ public void setUp() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Do you need `@Ignore` annotation on the test method ?

Reply:
Sorry my bad